Doc/library/multiprocessing.rst

```diff
@@ -377,32 +377,37 @@ However, if you really do need to use some shared data then
    :class:`list`, :class:`dict`, :class:`~managers.Namespace`, :class:`Lock`,
    :class:`RLock`, :class:`Semaphore`, :class:`BoundedSemaphore`,
    :class:`Condition`, :class:`Event`, :class:`Barrier`,
-   :class:`Queue`, :class:`Value` and :class:`Array`.  For example, ::
+   :class:`Queue`, :class:`Value`, :class:`Array` and :class:`set`.  For example, ::
```

Comment:
I forgot to mention here, but maybe it's better to put the mention to `set` closer to the mention of `list` and `dict`?